docs/http_api_swagger.md

```diff
@@ -396,6 +396,23 @@ Status: Internal Server Error
 
 
 
+### <span id="event-info"></span> event-info
+
+
+  
+
+
+
+**Properties**
+
+| Name | Type | Go type | Required | Default | Description | Example |
+|------|------|---------|:--------:| ------- |-------------|---------|
+| message | string| `string` |  | | Message describe the event which has occured. |  |
+| reason | string| `string` |  | | Reason is single work description of the subject of the event. |  |
```

Comment:
s/work/word

the reasons should be predefined in the system, so we can refer the user to documentation about the meaning of each event. 
What are the possibilities for `reason` and can it be documented (in a later stage we'd might be able to recommend on steps to debug those issues)

Reply:
Yes, there may be more events generated, where podman produce errors, currently we produce it only for pod start.